docs/containerd-env-setup.md

```diff
@@ -148,29 +150,6 @@ Nydus snapshotter has been supported by [nerdctl](https://github.com/containerd/
 $ sudo nerdctl --snapshotter nydus run --rm -it localhost:5000/ubuntu-nydus:latest bash
 ```
 
-## Try Nydus with `ctr-remote`
```

Comment:
If we'd to avoid `ctr-remote`, we should add another section to introduce `nerdctl`

Reply:
oh, please ignore it